src/kete_core/src/frames/definitions.rs

```diff
@@ -47,12 +47,24 @@ pub enum Frame {
 impl From<Frame> for i32 {
     fn from(value: Frame) -> Self {
         match value {
-            Frame::Unknown(_) => 0,
             Frame::Equatorial => 1,
             Frame::Ecliptic => 2,
             Frame::FK4 => 3,
             Frame::Galactic => 4,
             Frame::EclipticNonInertial(..) => 5,
+            Frame::Unknown(_) => 0,
+        }
+    }
+}
+
+impl From<i32> for Frame {
+    fn from(value: i32) -> Self {
+        match value {
+            1 => Frame::Equatorial,
+            2 => Frame::Ecliptic,
+            3 => Frame::FK4,
+            4 => Frame::Galactic,
+            i => Frame::Unknown(i as usize),
```

Comment:
Noninertial frames cannot be represented as an int, so get thrown into the unknown bucket. Above they are recorded as a 5 just for reference, but never used.